src/resources/views/crud/inc/form_save_buttons.blade.php

```diff
@@ -22,3 +22,74 @@
 
     <a href="{{ $crud->hasAccess('list') ? url($crud->route) : url()->previous() }}" class="btn btn-default"><span class="fa fa-ban"></span> &nbsp;{{ trans('backpack::crud.cancel') }}</a>
 </div>
+
+@push('after_scripts')
+<script>
+    function checkFormValidity(form) {
+        if (!form[0].checkValidity || form[0].checkValidity()) {
```

Comment:
Hmm... Wouldn't this return ```true``` always? If the condition is ```if (!form[0].checkValidity || form[0].checkValidity())``` wouldn't this be true at all times? Idk.

Reply:
From my understanding the first checkValidity checks if checkValidity() is supported in browser, the second actually runs the validation. Maybe I misinterpreted the docs. But my search lead me that way.

Reply:
Ok got it, thanks!